packages/neuron-wallet/tests/database/address/balance.test.ts

```diff
@@ -0,0 +1,114 @@
+import AddressEntity, { AddressVersion } from '../../../src/database/address/entities/address'
+import { AddressType } from '../../../src/models/keys/address'
+import initConnection, { getConnection } from '../../../src/database/address/ormconfig'
+import AddressDao, { Address } from '../../../src/database/address/dao'
+
+describe('balance', () => {
+  beforeAll(async () => {
+    await initConnection()
+  })
+
+  afterAll(async () => {
+    await getConnection().close()
+  })
+
+  beforeEach(async () => {
+    const connection = getConnection()
+    await connection.dropDatabase()
+    await connection.synchronize()
+  })
+
+  const generateAddress = (liveBalance: string | bigint, sentBalance: string | bigint, pendingBalance: string | bigint) => {
+    const addr = Math.round(Math.random() * 100000000).toString()
+    const address: Address = {
+      walletId: '1',
+      address: addr,
+      path: "m/44'/309'/0'/0/0",
+      addressType: AddressType.Receiving,
+      addressIndex: 0,
+      txCount: 0,
+      liveBalance: liveBalance.toString(),
+      sentBalance: sentBalance.toString(),
+      pendingBalance: pendingBalance.toString(),
+      balance: '0',
+      blake160: '0x36c329ed630d6ce750712a477543672adab57f4c',
+      version: AddressVersion.Testnet,
+    }
+    return address
+  }
+
+  it('balance = live + sent - pending', async () => {
+    const address = generateAddress('1000', '100', '300')
+    const addrs: AddressEntity[] = await AddressDao.create([address])
+    const addr = addrs[0]
+    expect(addr.balance()).toEqual((1000 + 100).toString())
+  })
+
+  it('balance is right in interface', async () => {
```

Comment:
```suggestion
  it('balance returned by the interface method', async () => {
```

Reply:
This test this focus on `balance` is right or not, not returned or not

Reply:
> This test this focus on `balance` is right or not, not returned or not

The description is a bit confusing, `balance is right in interface` means `the balance field is listed in the interface exactly`, try to use a better description, such as `the balance returned by the interface method is correct`